cirq-core/cirq/ops/control_values.py

```diff
@@ -40,16 +51,18 @@ def __and__(self, other: 'AbstractControlValues') -> 'AbstractControlValues':
         """
 
     @abc.abstractmethod
-    def _expand(self) -> Iterator[Tuple[Any, ...]]:
-        """Returns the control values tracked by the object."""
+    def _expand(self) -> Iterator[Tuple[int, ...]]:
+        """Returns a plain sum of product representation of the values instead
+        of the (possibly compressed) internal representation.
+        """
```

Comment:
nit: Either convert to a one-liner docstring or follow the guidance for multi-line docstrings described above. 
```suggestion
        """Expands the (possibly compressed) internal representation into a sum of products representation."""
```